src/components/MovieGrid/MovieGrid.tsx

```diff
@@ -3,22 +3,30 @@ import "./MovieGrid.scss"
 import { useState } from "react";
 
 import Counter from "../Counter/Counter"
-import MovieCard from "../MovieCard/MovieCard";
+import { MovieCard } from "../MovieCard/MovieCard";
 
+import Image from "../MovieCard/Bitmap.png";
 
 const MovieGrid = () => {
     const [counterInitialValue, setCounterInitialValue] = useState<number>(0);
+    const [movieData, setMovieData] = useState({
+        imageUrl: Image,
+        movieName: 'Example Movie',
+        releaseYear: '2022',
+        genres: ['Action', 'Drama'],
+    });
 
     return (
         <section className="movie-grid">
-            <Counter initialValue={counterInitialValue}/>
+            <Counter initialValue={counterInitialValue} />
             <div className="movie-cards-container">
-                <MovieCard/>
-                <MovieCard/>
-                <MovieCard/>
-                <MovieCard/>
-                <MovieCard/>
-                <MovieCard/>
+                <MovieCard {...movieData} onClick={() => console.log("card clicked")}/>
+                <MovieCard {...movieData} onClick={() => console.log("card clicked")}/>
+                <MovieCard {...movieData} onClick={() => console.log("card clicked")}/>
+                <MovieCard {...movieData} onClick={() => console.log("card clicked")}/>
+                <MovieCard {...movieData} onClick={() => console.log("card clicked")}/>
+                <MovieCard {...movieData} onClick={() => console.log("card clicked")}/>
+                <MovieCard {...movieData} onClick={() => console.log("card clicked")}/>
```

Comment:
looks like loop? :)

Reply:
it was test data, replaced with array, thank you for your comment.